hoomd/mpcd/ReversePerturbationUtilities.h

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2009-2024 The Regents of the University of Michigan.
+// Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+/*!
+ * \file ReversePerturbationUtilities.h
+ * \brief Helper functions for Reverse perturbation
+ */
+
+#ifndef AZPLUGINS_REVERSE_PERTURBATION_UTILITIES_H_
+#define AZPLUGINS_REVERSE_PERTURBATION_UTILITIES_H_
+
+#ifdef NVCC
+#define HOSTDEVICE __host__ __device__ __forceinline__
+#else
+#define HOSTDEVICE
+#endif
+
```

Comment:
I think you probably need to `#include "hoomd/HOOMDMath.h`, and it is lucky if this dependency is already met before this header gets included.